src/test/java/edu/cornell/kfs/krad/service/impl/CuAttachmentServiceImplTest.java

```diff
@@ -150,44 +212,45 @@ public void createAttachment() throws Exception {
         Assert.assertEquals("txt", createdAttachment.getAttachmentMimeTypeCode());
         Assert.assertEquals("txt", createdAttachment.getAttachmentTypeCode());
     }
+    
+    @Test
+    public void createAttachmentWithOutVirus() throws Exception {
```

Comment:
Is this scenario already being handled by the test case in this class's "createAttachment" test method?

Reply:
The other tests didn't drop into the virus scan's pass result.  This test does that.  I think createAttachment works around the scan code

Reply:
Thanks for checking. Now I see that the test method above this one is using the "dummy" AV service instead, so your new method is indeed testing a new scenario.